account_move_base_import/models/account_move.py

```diff
@@ -274,7 +275,32 @@ def _prepare_insert(self, move, cols):
         move_copy = move
         for k, col in move_copy.items():
             if k in cols:
-                move_copy[k] = self._fields[k].convert_to_column(col, None)
+                # In v11, the record is needed to call convert_to_column on a
+                # Monetary field because it must find the currency to apply the
+                # proper rounding. So we mimic the call here as best as we can.
```

Comment:
Question: if a record is needed, could we use `new()` to get a fake record to retrieve the monetary value and avoid this workaround?